src/logic/install.ts

```diff
@@ -484,7 +476,12 @@ export async function getProfileFromStore(
       logger.info('fetchProfileAst', profileIdStr);
     } catch (error) {
       logger.warn('fetchProfileAstFailed', profileIdStr);
-      //We try to parse profile on our own
+      //We try to get source and parse profile on our own
+      const profile = await fetchProfile(profileId, version, {
+        tryToAuthenticate: options?.tryToAuthenticate,
+      });
+      logger.info('fetchProfileSource', profileIdStr);
+
```

Comment:
issue: I am not sure about this. because what it can cause is that locally will everything work. But once deployed to production it won't because on production we will not get profile ast.

Reply:
I think better is to just fail and ask to run install again if it was problem to fetch AST. If validation failed, then we are in bigger trouble, because we don't have good mechanism how to check valid ast for OneSDK it will be used with.